packages/deployments/contracts/contracts/messaging/WatcherManager.sol

```diff
@@ -29,17 +29,17 @@ contract WatcherManager is ProposedOwnable {
    * @dev Owner can enroll a watcher (abilities are defined by inheriting contracts)
    */
   function addWatcher(address _watcher) external onlyOwner {
-    require(!watchers[_watcher], "already watcher");
-    watchers[_watcher] = true;
+    require(!isWatcher[_watcher], "already watcher");
+    isWatcher[_watcher] = true;
     emit WatcherAdded(_watcher);
```

Comment:
We don't need to check `address(0)` when adding new watcher?

Reply:
we could, but it would be impossible to make that the `msg.sender`. 

should still add the check